ush/config_defaults.sh

```diff
@@ -373,20 +373,86 @@ FCST_LEN_HRS="24"
 # Set METplus parameters.  Definitions:
 #
 # MODEL: 
+# String that specifies a descriptive name for the model being verified.
+#
 # MET_INSTALL_DIR:
+# Location to top-level directory of MET installation.
+#
 # METPLUS_PATH:
-# VX_CONFIG_DIR:
-# METPLUS_CONF:
-# MET_CONFIG:
+# Location to top-level directory of METplus installation.
+#
 # CCPA_OBS_DIR:
+# User-specified location of top-level directory where CCPA hourly
+# precipitation files used by METplus are located. This parameter needs
+# to be set for both user-provided observations and for observations 
+# that are retrieved from the NOAA HPSS (if the user has access) via
+# the get_obs_ccpa_tn task (activated in workflow by setting 
+# RUN_TASK_GET_OBS_CCPA="TRUE"). METplus is configured to verify 01-, 
+# 03-, 06-, and 24-h accumulated precipitation using hourly CCPA files. 
+# METplus configuration files require the use of predetermined directory 
+# structure and file names. Therefore, if the CCPA files are user 
+# provided, they need to follow the anticipated naming structure: 
+# {YYYYMMDD}/ccpa.t{HH}z.01h.hrap.conus.gb2, where YYYY is the 4-digit 
+# valid year, MM the 2-digit valid month, DD the 2-digit valid day of 
+# the month, and HH the 2-digit valid hour of the day. In addition, a 
+# caveat is noted for using hourly CCPA data. There is a problem with 
+# the valid time in the metadata for files valid from 19 - 00 UTC (or 
+# files  under the '00' directory). The script to pull the CCPA data 
+# from the NOAA HPSS has an example of how to account for this as well
+# as organizing the data into a more intuitive format: 
+# regional_workflow/scripts/exregional_get_ccpa_files.sh. When a fix
+# is provided, it will be accounted for the exregional_get_ccpa_files.sh
```

Comment:
Very small typo - it will be accounted for "IN" the exregional_get_ccpa_files.sh.